components/modal/modal.css

```diff
@@ -0,0 +1,18 @@
+.modal{
+    padding: 0.75rem 1rem;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+    width: fit-content;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Just a recommendation rohan , if you are using flex try to use it to center the modal not position's

Reply:
we need modal on top of the other elements so for that I think justify content or align items property will not work...I should use position as fixed and fixed that now. Thanks for your review. You can check now.